src/Backend/Core/Js/backend.js

```diff
@@ -359,9 +359,6 @@ jsBackend.ckeditor =
         filebrowserImageUploadUrl: null,
         filebrowserFlashUploadUrl: null,
 
-        // uploading drag&drop images, see http://docs.ckeditor.com/#!/guide/dev_file_upload
```

Comment:
I think we can deprecate it but not sure if we should remove it completely yet, since that is a BC break

Reply:
@carakas Ok, I'll drop that commit.